app/components/data/shared/cloud-file-picker/cloud-file-picker.component.ts

```diff
@@ -28,6 +28,12 @@ export class CloudFilePickerComponent implements ControlValueAccessor, OnChanges
      */
     @Input() public containerId: string;
 
+    /**
+     * Passed in if we want to client side filter the results to only show
+     * files that end with this suffix.
+     */
+    @Input() public suffixFilter: string;
```

Comment:
I would maybe here instead have the same interface as openDialog from electron 

```js
filters: [
    { name: 'All Files', extensions: ['*'] },
    { name: 'Images', extensions: ['jpg', 'png', 'gif'] },
    { name: 'Movies', extensions: ['mkv', 'avi', 'mp4'] }
],
````